assets/i18n/en.json

```diff
@@ -73,7 +73,26 @@
     "I18N_ADD_NEW_SYLLABUS_ITEMS": "New Syllabus Items",
     "I18N_ADD_SYLLABUS_DESCRIPTION_TEXT": "Add skills or stories to your syllabus to automatically send them to your learners.",
     "I18N_ADD_SYLLABUS_SEARCH_PLACEHOLDER": "Search eg. Story,  Physics, English",
+    "I18N_ANDROID_PAGE_AVAILABLE_FOR_DOWNLOAD_TEXT": "available for download.",
+    "I18N_ANDROID_PAGE_BETA_DESCRIPTION": "A beta version of the Android app is now available in <strong>English</strong>,<br><strong>Brazilian</strong>, <strong>Portuguese</strong>, and <strong> Swahili</strong>. The app is free, and no<br>subscription is needed.",
+    "I18N_ANDROID_PAGE_CONSENT_CHECKBOX_LABEL": "You confirm that you are over the age of 18 or that you have consent and approval from your legal parent or guardian.",
+    "I18N_ANDROID_PAGE_EMAIL_FIELD_LABEL": "Email address",
+    "I18N_ANDROID_PAGE_FEATURE_SUBTEXT_1": "And try our helpful hints to guide you along",
+	"I18N_ANDROID_PAGE_FEATURE_SUBTEXT_2": "Play lessons even without an Internet connection.",
+	"I18N_ANDROID_PAGE_FEATURE_SUBTEXT_3": "For now, the app is available in <strong>English</strong>, <strong>Brazilian</strong><br><strong>Portuguese</strong>, and<strong> Swahili</strong>.<br><br>More languages will be added soon!",
+	"I18N_ANDROID_PAGE_FEATURE_SUBTEXT_4": "Create and maintain up to 10 profiles on one device.",
+	"I18N_ANDROID_PAGE_FEATURE_TEXT_1": "Learn through engaging stories",
+	"I18N_ANDROID_PAGE_FEATURE_TEXT_2": "Learn anytime, anywhere",
+	"I18N_ANDROID_PAGE_FEATURE_TEXT_3": "Learn in your language",
+	"I18N_ANDROID_PAGE_FEATURE_TEXT_4": "Switch between learners",
```

Comment:
Why use tabs here?

Reply:
Ah this was not intentional, seems to be due to a copy-paste from qqq.json (where we seem to use tabs.) Done.